docs/Getting Started/Fedora/Fedora Root On ZFS.rst

```diff
@@ -0,0 +1,753 @@
+   ``mount`` or ``zfs mount``). This situation is an exception because of
+   ``canmount=noauto``.
+
+#. Create datasets::
+
+     zfs create                                 rpool/home
+     zfs create -o mountpoint=/root             rpool/home/root
+     zfs create -o canmount=off                 rpool/var
+     zfs create -o canmount=off                 rpool/var/lib
+     zfs create                                 rpool/var/log
+     zfs create                                 rpool/var/spool
+
+   The datasets below are optional, depending on your preferences and/or
+   software choices.
+
+   If you wish to exclude these from snapshots::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/cache
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/tmp
+     chmod 1777 /mnt/var/tmp
+
+   If this system will use Docker (which manages its own datasets &
+   snapshots)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/docker
+
+   If this system will use NFS (locking)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/nfs
+
+   A tmpfs is recommended later, but if you want a separate dataset for
+   ``/tmp``::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/tmp
+     chmod 1777 /mnt/tmp
+   
+   Note that the reason why we are not fully seperating everything like we did in Ubuntu is because dnf will fail to install or update certain packages if we create too many datasets.
+
+   The primary goal of this dataset layout is to separate the OS from user
+   data. This allows the root filesystem to be rolled back without rolling
+   back user data.
+
+   If you do nothing extra, ``/tmp`` will be stored as part of the root
+   filesystem. Alternatively, you can create a separate dataset for ``/tmp``,
+   as shown above. This keeps the ``/tmp`` data out of snapshots of your root
+   filesystem. It also allows you to set a quota on ``rpool/tmp``, if you want
+   to limit the maximum space used. Otherwise, you can use a tmpfs (RAM
+   filesystem) later.
+
+#. Copy the LiveCD to your HDD/SDD::
+
+     rsync -avxHASX / /mnt/
+   
+   It is important to not forget the trailing /.
+   This command copies the LiveCD to our new zfs datasets and this is the only way I have found to reliably install and boot Fedora Workstation
+
+Step 4: System Configuration
+----------------------------
+
+#. Configure the hostname:
+
+   Replace ``HOSTNAME`` with the desired hostname::
+
+     echo HOSTNAME > /mnt/etc/hostname
+     vi /mnt/etc/hosts
+
+   .. code-block:: text
+
+     Add a line:
+     127.0.1.1       HOSTNAME
+     or if the system has a real name in DNS:
+     127.0.1.1       FQDN HOSTNAME
+
+   **Hint:** Use ``nano`` or ``vim`` if you find ``vi`` confusing.
+
+#. Configure the network interface:
+
+   Find the interface name::
+
+     ip addr show
+
+   Adjust ``NAME`` below to match your interface name::
+
+     vi /mnt/etc/network/interfaces.d/NAME
```

Comment:
it is 2020, it's time to stop using old config files and either use `nmconnection` in `/etc/NetworkManager/system-connections/` when using NetworkManager which is default or `network` unit files in `/etc/systemd/network` when using `systemd-networkd`

Reply:
This isn't so much an issue of _old_ config files as this is a Debian-ism that hasn't been ported to Fedora's way yet.